src/detector/detector.py

```diff
@@ -480,15 +260,16 @@ def start_capture(self, **kwargs) -> MCastResponse:
             return EmptyResponse()
         self.add_status_message(severity="info", message=f"{client_identifier} is starting capture.")
 
-        self._capture = cv2.VideoCapture(self._capture_device_id,  cv2.CAP_DSHOW)
+        # CAP_DSHOW
+        self._capture = cv2.VideoCapture(self._capture_device_id)
```

Comment:
Similarly to above. If the same code is needed in a few different places, it might make sense to make a private function (name contains a leading underscore `_`)